tests/phpunit/tests/Story_Post_Type.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+/**
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace Google\Web_Stories\Tests;
+
+class Story_Post_Type extends \WP_UnitTestCase {
```

Comment:
Due to this new class, `\Google\Web_Stories\Tests\Story_Renderer` needs to be updated to reference the correct `Story_Post_Type` class.

```
Fatal error: Cannot use Google\Web_Stories\Story_Post_Type as Story_Post_Type because the name is already in use in web-stories-wp/tests/phpunit/tests/Story_Renderer.php on line 21
```